onnxmltools/convert/coreml/operator_converters/neural_network/Embed.py

```diff
@@ -7,8 +7,8 @@
 import numpy as np
 from .Scale import deduce_broadcast_axis_and_shape
 from .....proto import onnx_proto
-from ....common._apply_operation import apply_add, apply_reshape, apply_cast
-from ....common._registration import register_converter
+from onnxtk.common._apply_operation import apply_add, apply_reshape, apply_cast
```

Comment:
As we discussed today, we should keep the common folder and in this folder we can import the types and functions from onnxtk, so we can avoid a lot of changes in the existing converter functions and had a better backward compatibility.

Reply:
Updated in latest version!